etc/sysconfig/pacemaker.in

```diff
@@ -275,6 +275,16 @@
 #
 # Default: PCMK_ipc_buffer="131072"
 
+# PCMK_cib_timeout (Advanced Use Only)
+#
+# Specify the timeout (in seconds) for the controller to use with CIB
+# operations.
+#
+# The default value is computed as the maximum of 30 and
+# (10 * ((number of nodes in the cluster) + 1)).
+#
+# Default: PCMK_cib_timeout=""
+
```

Comment:
> This is used to cap the computed value, rather than the computed value being the default of this.

It's whichever one is larger. It doesn't look like this caps the computed value.
```
calculated_timeout = QB_MAX(calculated_timeout, env_timeout);
```

My description is also wrong though. The actual timeout is effectively `max(30, PCMK_cib_timeout, calculated_timeout)`.

---

Yeah, I think a cluster option would be fine. If so, can we drop the env var, or keep for compatibility? It was never documented.

Not sure about `cluster-delay`. I agree that it's documented as being for inter-node communication, and it's used to set a `network_delay` member. However, it's not clear to me that it applies to "delays from any source." It is used as the controller's default timeout for cluster actions and resource actions (at least the latter may be local), if the configured timeout is <= 0.

This use case is for IPC with the local CIB manager via `controld_globals.cib_conn`, rather than inter-node. My feelings aren't strong though.

Reply:
Bah I read this one twenty times and still interpreted it wrong. Let's leave it undocumented and deprecate it via comment. I don't think we need it at all (even as a cluster option). With 2 or fewer active nodes, we'll use 30s. With more than 2, the formula already seems excessive to me -- there should never be a reason to raise it even higher.